nautobot/core/admin.py

```diff
@@ -48,3 +50,9 @@ class ConfigAdmin(ConstanceAdmin):
 
 admin_site.unregister([Config])
 admin_site.register([Config], ConfigAdmin)
+
+
+class BaseAdmin(ModelAdmin):
+    formfield_overrides = {
+        models.ManyToManyField: {"widget": StaticSelect2Multiple},
+    }
```

Comment:
I wonder if it would also be more consistent if we make `models.ForeignKey` use a `StaticSelect2` widget? I'll pull down your branch and give it a try.

Reply:
Good call